packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -756,6 +756,7 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
             // if not yet contributed by Monaco, check runtime css variables to learn
             { id: 'selection.background', defaults: { dark: '#217daf', light: '#c0dbf1' }, description: 'Overall border color for focused elements. This color is only used if not overridden by a component.' },
             { id: 'icon.foreground', defaults: { dark: '#C5C5C5', light: '#424242', hc: '#FFFFFF' }, description: 'The default color for icons in the workbench.' },
+            { id: 'contrastBorder', defaults: { hc: '#6FC3DF' }, description: 'An extra border around elements to separate them from others for greater contrast.' },
```

Comment:
@jbicker are you sure that it is not provided by Monaco already? Did you check runtime variables with hc theme on?

Reply:
I will add a runtime error logging that we avoid redeclaring Monaco colors.

Reply:
Verified. It is not.

Reply:
I re-verified. It is. It is removed here with the next commit.